--- db-5.3.15/src/env/env_config.c.string_whitespace~	2011-12-20 02:00:51.000000000 +0100
+++ db-5.3.15/src/env/env_config.c	2012-02-17 00:13:06.127978208 +0100
@@ -722,6 +722,9 @@ __config_scan(input, argv, descptr)
 			if (*descptr != NULL &&
 			    (*descptr)->type == CFG_STRING) {
 				count++;
+				/* Make sure to remove any leading whitespace */
+				if (isblank(*input))
+				    strsep(&input, " \t");
 				*ap++ = input;
 				break;
 			}
